### ISSUE TITLE
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`

### ISSUE THREAD

**Issue by AlphaPlusTT** [JDAI-CV/fast-reid#501]
/home/111/anaconda3/envs/fastreid/lib/python3.7/site-packages/torch/optim/lr_scheduler.py:123: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate

**Comment by L1aoXingyu** [comment]
fix it on https://github.com/JDAI-CV/fast-reid/commit/8f8cbf94116b365f34e4971ac68f0fb01d33aab0
